docs/reference/contributing/guidelines/guidelines.md

```diff
@@ -19,3 +19,11 @@ If so, you can request to become an organization member, but you must first ensu
 - All users must have 2 Factor Authentication enabled.
 - Arm staff must have their Name, Company (Arm), Location and Arm email address publicly visible.
 - All others should have their Name and Company visible. Entering your Location will help us interpret response times according to time zones.
+
+### New repository
+
+Each repository must contain:
+
+- `CONTRIBUTING.md` file (similar to [Mbed OS contributing](https://github.com/ARMmbed/mbed-os/blob/master/CONTRIBUTING.md)).
+- `LICENSE` file - the full license text or overview of every license in the repo with links.
```

Comment:
Might not be a question for this PR, but do we need to specify which kind of licenses should be used?

Reply:
Shall we add a link here to license page ?

Reply:
Oh, yes!

Reply:
`* Licenses: Licenses should comply with the [licenses described here](../contributing/license.html).` is above in the first section, should be sufficient for this page